src/sacn_mock/common.c

```diff
@@ -41,4 +42,8 @@ void sacn_reset_all_fakes(void)
   sacn_common_reset_all_fakes();
   sacn_data_loss_reset_all_fakes();
   sacn_sockets_reset_all_fakes();
+
+#ifdef TESTING_CPP_MERGER
```

Comment:
I don't think this #ifdef is necessary - prefer to select which fake and real modules to bring in in a granular way in the unit test executable, and call the individual module reset functions as necessary in the test setup.